Javascript/q42.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>q42</title>
+
+  <style>
+    body {
+      font-family: Verdana, Geneva, Tahoma, sans-serif;
+    }
+
+    .flex {
+      display: flex;
+      flex-wrap: wrap;
+      width: 300px;
+      gap: 10px;
+    }
+
+    .box {
+      width: 200px;
+    }
+
+    button {
+      margin-top: 5px;
+    }
+  </style>
+
+  <script>
+    function myFunction(){
+      let min = Number(document.getElementById("demo0").value);
+      let max = Number(document.getElementById("demo1").value);
+      let x = Math.round(Math.random() * (max - min + 1) + min);
+      document.querySelector("#demo").innerHTML = " " + x;
+    }
+  </script>
+</head>
+
+<body>
+  <div class="flex">
+    <div>Min limit</div>
+    <input id="demo0" type="number" class="box" style="margin-left: 4px;"></input>
+    <div>Max limit</div>
+    <input id="demo1" type="number" class="box"></input>
+  </div>
+  <button onclick="myFunction()">Generate random</button>
+  <div>Generated Num:<span id="demo"></span></div>
```

Comment:
Please fix all of these `id` values. `demo` is not a very useful or descriptive name.

Reply:
Please fix them in the `input` elements above as well

Reply:
fixed